src/components/data-exploration/embedding/DynamicVitessceEmbedding.jsx

```diff
@@ -0,0 +1,6 @@
+// Due to a bug with dynamic imports and webpack, we need to create
+// a wrapper component to import the Scatterplot component.
+// See https://github.com/webpack/webpack/issues/13865
```

Comment:
Very weird bug. So importing it here in the wrapper works, but not in other places? How?

Reply:
Yes, it is super weird. I think that the bug is that you can't both import the real thing (`@vitessce/heatmap`) and use dynamic imports in one place -- you have to do it one by one, because webpack doesn't like they syntax of it written all at once. This is why it works when you do it with a wrapper. This is my understanding.